continuous_integration/recipe/meta.yaml

```diff
@@ -11,40 +10,48 @@ source:
   git_url: ../..
 
 build:
-  noarch: python
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  skip: true   # [py2k]
+  entry_points:
+    - dask-sql-server = dask_sql.server.app:main
+    - dask-sql = dask_sql.cmd:main
+  string: py{{ py_version.replace(".", "") }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```

Comment:
Forgot about this - is there a clean way to get a Python version for the package string from the build variant?

Reply:
Can just replace with `python` instead of `py_version`. Though maybe you have already done this :)